common/app/model/meta.scala

```diff
@@ -21,45 +22,26 @@ import play.twirl.api.Html
 
 object Commercial {
 
-  private def make(metadata: MetaData, tags: Tags, maybeApiContent: Option[contentapi.Content]): model.Commercial = {
-
-    val section = Some(metadata.sectionId)
-
-    val isInappropriateForSponsorship =
-      maybeApiContent exists (_.fields.flatMap(_.isInappropriateForSponsorship).getOrElse(false))
+  def make(tags: Tags, apiContent: CapiContent): model.Commercial = {
+    val isInappropriateForSponsorship: Boolean = apiContent.fields.exists(_.isInappropriateForSponsorship.contains(true))
 
     model.Commercial(
-      tags,
-      metadata,
       isInappropriateForSponsorship,
       hasInlineMerchandise = DfpAgent.hasInlineMerchandise(tags.tags)
     )
   }
 
-  def make(metadata: MetaData, tags: Tags, apiContent: contentapi.Content): model.Commercial = {
-    make(metadata, tags, Some(apiContent))
-  }
-
-  def make(metadata: MetaData, tags: Tags): model.Commercial = {
-    make(metadata, tags, None)
-  }
-
-  def make(section: Section): model.Commercial = {
+  def make: model.Commercial = {
```

Comment:
This could be a `val` called `empty` or something, but perhaps it's not even needed.